engine/swarm/admin_guide.md

```diff
@@ -231,29 +247,47 @@ you demote or remove a manager
 ## Recover from disaster
 
 Swarm is resilient to failures and the swarm can recover from any number
-of temporary node failures (machine reboots or crash with restart).
+of temporary node failures (machine reboots or crash with restart) or other
+transient errors. However, a swarm cannot automatically recover if it loses a
+quorum.
 
-In a swarm of `N` managers, there must be a quorum of manager nodes greater than
-50% of the total number of managers (or `(N/2)+1`) in order for the swarm to
-process requests and remain available. This means the swarm can tolerate up to
-`(N-1)/2` permanent failures beyond which requests involving swarm management
-cannot be processed. These types of failures include data corruption or hardware
-failures.
+In a swarm of `N` managers, a quorum (a majority) of manager nodes must always
+be available. For example, in a swarm with 5 managers, a minimum of 3 must be
+operational and in communication with each other. In other words, the swarm can
+tolerate up to `(N-1)/2` permanent failures beyond which requests involving
+swarm management cannot be processed. These types of failures include data
+corruption or hardware failures.
+
+### Recovering from losing the quorum
+
+If you lose the quorum of managers, you cannot administer the swarm. If you have
+lost the quorum and you attempt to perform any management operation on the swarm,
+an error occurs:
```

Comment:
@stevvooe you think we should repeat that tasks remain to run (https://github.com/docker/docker.github.io/pull/297/files#r88358814)? Train of thought; If the lost quorum is temporary, but all tasks are still running, there may not be a direct need to try to recover.